Formula/wb32-dfu-updater_cli.rb

```diff
@@ -0,0 +1,30 @@
+class Wb32DfuUpdaterCli < Formula
+  desc "USB programmer"
+  homepage "https://github.com/WestberryTech/wb32-dfu-updater"
+  url "https://github.com/WestberryTech/wb32-dfu-updater/archive/refs/tags/1.0.0.tar.gz"
+  sha256 "2b1c5b5627723067168af9740cb25c5c179634e133e4ced06028462096de5699"
+  license "Apache-2.0"
+  head "https://github.com/WestberryTech/wb32-dfu-updater.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "libusb"
+
+  def install
+    args = std_cmake_args
+
+    libusb = Formula["libusb"]
+    args << "-DLIBUSB_INCLUDE_DIRS=#{libusb.opt_include}/libusb-1.0"
+    args << "-DLIBUSB_LIBRARIES=#{libusb.opt_lib/shared_library("libusb-1.0")}"
```

Comment:
Is cmake not able to find these normally?

Reply:
There may be some differences for different systems.

Reply:
This is how I do it on Windows for different compilers, Macos doesn't need to do this, which I've removed.